src/sentry/event_manager.py

```diff
@@ -2060,13 +2069,24 @@ def _get_severity_score(event: Event) -> float | None:
                 except MaxRetryError as e:
                     logger.warning(
```

Comment:
@lobsterkatie could we also modify this log message to be a static string and pass the another variables in `extra`? The way our logging is set up it's hard to find these log messages with dynamic strings on the event.

Reply:
My thought here was, these messages serve two purposes: record-keeping and information-supplying when running/testing locally. On the record keeping side, `extra` for all the messages now has `op: "event_manager._get_severity_score"`, specifically so it'll be easy to find them all. (See second bullet point above. [This query](https://console.cloud.google.com/logs/query;query=labels.name%20%3D%20%22sentry.events%22%0AjsonPayload.op%20%3D%20%22event_manager._get_severity_score%22%0A;summaryFields=:false:32:beginning;cursorTimestamp=2023-09-14T19:52:12.669119956Z;duration=P1D?organizationId=225313812765&project=internal-sentry) is currently empty, but once this is merged it will include all the logs from `_get_severity_score`, regardless of the outcome.) On the info-supplying side, only the message is displayed (not `extra`) so if we want those to be useful, the dynamic data has to be in there.

In the interests of moving things along, I'm going to merge this for now. If we fix the bug and still want to keep this logging, we can revisit how the data is presented/stored.